src/mixxx.cpp

```diff
@@ -1459,6 +1463,11 @@ void MixxxMainWindow::initActions()
     }
 }
 
+void MixxxMainWindow::changeWindowTitle(QString title)
+{
+    this->setWindowTitle(title);
```

Comment:
Ah no, we need it. Please append the original title:  " - Mixxx VERSION" 
This seams to be a common practice in Gedit and Firefox.